pyanaconda/modules/storage/devicetree/model.py

```diff
@@ -353,23 +358,31 @@ def protect_devices(self, protected_names):
         # Update the list.
         self.protected_devices = protected_names
 
-    def _mark_protected_device(self, device):
+    def _mark_protected_device(self, device, ancestors=False):
```

Comment:
Could we please name the boolean to make it's obvious it's not a list of ancestors. I would suggest `protect_ancestors` or `include_ancestors`?

Reply:
Updated